src/crypto/public_key/elliptic/brainpool/brainpoolP256r1.ts

```diff
@@ -0,0 +1,24 @@
+import { createCurve } from '@noble/curves/_shortw_utils';
+import { sha256 } from '@noble/hashes/sha256';
+import { Field } from '@noble/curves/abstract/modular';
+
+// brainpoolP256r1: https://datatracker.ietf.org/doc/html/rfc5639#section-3.4
+
+// eslint-disable-next-line new-cap
+const Fp = Field(BigInt('0xa9fb57dba1eea9bc3e660a909d838d726e3bf623d52620282013481d1f6e5377'));
```

Comment:
Can we use BigInt syntax directly, now? E.g.
```suggestion
const Fp = Field(0xa9fb57dba1eea9bc3e660a909d838d726e3bf623d52620282013481d1f6e5377n);
```
and so on (everywhere else too)

Reply:
`BigInt` syntax is more widely supported apparently: https://github.com/paulmillr/noble-curves/issues/22 .

Reply:
That issue is talking about their build environment, rather than any browsers. As far as I can tell, all browsers that support the BigInt() constructor also support the syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#browser_compatibility

Reply:
OK. I suppose we can always revisit this later and see whether BigInt notation is widely supported enough yet.